addons/medical_vitals/functions/fnc_updateOxygen.sqf

```diff
@@ -0,0 +1,83 @@
+#include "script_component.hpp"
+/*
+ * Author: Brett Mayson
+ * Update the oxygen levels
+ *
+ * Arguments:
+ * 0: The Unit <OBJECT>
+ * 1: Time since last update <NUMBER>
+ * 2: Sync value? <BOOL>
+ *
+ * ReturnValue:
+ * Current SPO2 <NUMBER>
+ *
+ * Example:
+ * [player, 1, false] call ace_medical_vitals_fnc_updateOxygen
+ *
+ * Public: No
+ */
+
+params ["_unit", "_deltaT", "_syncValue"];
+
+#define IDEAL_PPO2 0.255
+
+private _current = GET_SPO2(_unit);
+private _heartRate = GET_HEART_RATE(_unit);
+
+private _altitude = (getPosASL _unit) select 2;
```

Comment:
```suggestion
private _altitude = EGVAR(common,mapAltitude) + (getPosASL _unit) select 2;
```
some maps have an offset - ref https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_getMapData.sqf
we use it in some places but not everywhere
shouldn't hurt to include here if someone makes an Everest map 😄